packages/astro/test/reuse-port.test.js

```diff
@@ -0,0 +1,25 @@
+import { deepStrictEqual, ok } from 'node:assert'
+import { resolve } from 'node:path'
+import { test } from 'node:test'
+import { setFixturesDir, verifyReusePort } from '../../basic/test/helper.js'
+
+process.setMaxListeners(100)
```

Comment:
why do we need this?

Reply:
I think I copied from another test. Removing it.